src/solve/test.rs

```diff
@@ -1872,8 +1872,10 @@ fn inscope() {
 
         goal {
             InScope(Foo)
-        } yields {
+        } yields[SolverChoice::Default] {
             "No possible solution: no applicable candidates"
+        } yields[SolverChoice::SLG(MAX_SIZE)] {
+            "No possible solution: no answers"
```

Comment:
If you just make this `"No possible solution"` it will work for both solvers (we only check for a prefix).

Reply:
I didn't see this distinction being made in any of the other tests, so that's why I thought something wasn't entirely right here. 😛